docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -0,0 +1,200 @@
+1. Click **Create** to add a new repository.
+1. Enter the following:
+    - **Name**: turtles
+    - **Index URL**: https://rancher.github.io/turtles
+1. Wait until the new repository has a status of `Active`.
+1. In the left navigation menu, select **Apps > Charts**.
+1. Enter "turtles" into the search filter to find the Turtles chart.
+1. Click **Rancher Turtles - the Cluster API Extension**.
+1. Click **Install > Next > Install**.
+
+This process uses the default values for the Helm chart, which are good for most installations. If your configuration requires overriding some of these defaults, you can either specify the values during installation from the Rancher UI or you can [manually install the chart via Helm](./overview.md#via-helm-install). For details about available values, see the Rancher Turtles [Helm chart reference guide](https://turtles.docs.rancher.com/docs/reference-guides/rancher-turtles-chart/values).
+
+The installation may take a few minutes and once completed you will see the following new deployments in the cluster:
+- `rancher-turtles-system/rancher-turtles-controller-manager`
+- `rancher-turtles-system/rancher-turtles-cluster-api-operator`
+- `capi-system/capi-controller-manager`
+
+### Via Helm Install
+
+There are two ways to install Rancher Turtles with Helm, depending on whether you include the CAPI operator as a dependency:
+- [Install Rancher Turtles with CAPI Operator as a dependency](#install-rancher-turtles-with-cluster-api-capi-operator-as-a-helm-dependency).
+- [Install Rancher Turtles without CAPI Operator](#install-rancher-turtles-without-cluster-api-capi-operator-as-a-helm-dependency).
+
+The CAPI Operator is required for installing Rancher Turtles. You can choose whether you want to take care of this dependency yourself or let the Rancher Turtles Helm chart manage it for you. [Installing Turtles as a dependency](#install-rancher-turtles-with-cluster-api-capi-operator-as-a-helm-dependency) is simpler, but your best option depends on your specific configuration.
+
+The CAPI Operator allows for handling the lifecycle of CAPI providers using a declarative approach, extending the capabilities of `clusterctl`. If you want to learn more about it, you can refer to [Cluster API Operator book](https://cluster-api-operator.sigs.k8s.io/).
+
+#### Install Rancher Turtles with `Cluster API (CAPI) Operator` as a Helm dependency
+
+1. The `rancher-turtles` chart is available in https://rancher.github.io/turtles. This Helm repository must be added before installation:
+
+```bash
+helm repo add turtles https://rancher.github.io/turtles
+helm repo update
+```
+
+2. As mentioned before, installing Rancher Turtles requires the [CAPI Operator](https://github.com/kubernetes-sigs/cluster-api-operator). The Helm chart can automatically install it with a minimal set of flags:
+
+```bash
+helm install rancher-turtles turtles/rancher-turtles --version v0.5.0 \
```

Comment:
```suggestion
helm install rancher-turtles turtles/rancher-turtles --version <version> \
```

Similar to other areas in our docs, we should avoid listing specific versions, if possible (e.g. there are no version comparability limitations). It adds to maintenance or we end up with users installing older versions. E.g. `0.6.0` is available as of this moment.

Also, there's no `v` in the version.

```
helm search repo turtles
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
turtles/rancher-turtles 0.6.0           0.6.0           Rancher Turtles is an extension to Rancher that...
```